models/pathology_nuclei_segmentation_classification/large_files.yml

```diff
@@ -1,5 +1,5 @@
 large_files:
   - path: "models/model.pt"
-    url: "https://drive.google.com/uc?id=16CLRwqwxsToEmH-rSUOn2CSj_KmByGbp"
-    hash_val: "c00b643a4b4f9bdeb41ddf5a25cbea0a"
+    url: "https://drive.google.com/file/d/1Pznw-5AeoybEdjCrjmN-TzMHjWW0UjXn/view?usp=share_link"
```

Comment:
```suggestion
    url: "https://drive.google.com/uc?id=1Pznw-5AeoybEdjCrjmN-TzMHjWW0UjXn"
```

Reply:
have you guys verified the contours that you are generating.. after running hovernet sliding window inferer for some ROI?

Reply:
Hi @SachidanandAlle, I have verified it in jupyter notebook. But I didn't compare it with the results from the original repo and just had there is some inconsistency. Have created an issue to solve it.
https://github.com/Project-MONAI/MONAI/issues/5750
Thanks!

Reply:
Hi @drbeh, may I know why need to change the URL? Seems we already have the same format with the URLs from other bundles.
https://github.com/Project-MONAI/model-zoo/blob/fd95444a2feac5f76d9350a748deba5ba63311a2/models/brats_mri_segmentation/large_files.yml#L3

Reply:
Hi @SachidanandAlle, the inconsistency issue has been solved in https://github.com/Project-MONAI/MONAI/pull/5751. Do you have any other comments on this PR? Thanks!

Reply:
This way of representing a google drive link does not allow `gdown` to download it and instead it downloads the html page. Using `uc?id=` instead of `file/d/` can solve this issue.

@SachidanandAlle, how do you download the model weights for bundle when you are invoking it for a MOANILabel use case?

Reply:
Hi @SachidanandAlle ,

We updated the bundle with pretrained weights from internal ImageNet instead of the previous public ImageNet.
Could you please help download and have a try?

Thanks.

Reply:
Infer looks good.. let me see if I can finetune the model over this..

Reply:
basic train workflow is ok via monailabel but the final model.pt (best metric should be correctly placed for others to consume/refresh)

![image](https://user-images.githubusercontent.com/7339051/207995888-7c3df9eb-2451-4f5d-80bf-970531a3f98a.png)

Reply:
Hi @KumoLiu ,

I think you missed the `key_metric_filename` arg in the train config, please refer to:
https://github.com/Project-MONAI/model-zoo/blob/dev/models/spleen_ct_segmentation/configs/train.json#L255

Thanks.

Reply:
you can have any number of stages.. but final model which is ready for others to consume should be placed back into the correct `models` folder.  even inference scripts loads from models/mode.pt

https://github.com/KumoLiu/model-zoo/blob/hovernet-bundle/models/pathology_nuclei_segmentation_classification/configs/inference.json#L130

Reply:
Previous comment for reference: https://github.com/Project-MONAI/model-zoo/pull/245#discussion_r1043742493

Reply:
only this one issue.. otherwise all ok from myside.. to release the bundle.. verified for multi gpu as well

Reply:
Hi @SachidanandAlle, since we don't have validation data, the second stage reads the last epoch of the train, and the reference also reads the last epoch of the train of the second stage.
https://github.com/Project-MONAI/model-zoo/blob/700d1bdd4eca349ab06bd2cc75f1dff8247af911/models/pathology_nuclei_segmentation_classification/configs/train.json#L330-L331
Thanks!

Reply:
So you mean to say you can not produce models/model.pt once the training is finished?

Reply:
Hi @SachidanandAlle, Sorry for the confusion in the bundle, actually HoVerNet pipeline uses the last epoch model of the train as the final output model, instead of the best validation model. So you don't need to load the best validation key metric model, just use the `model.pt`.
And when you train stage 1,  we will automatically read the model from the first stage's fold. Actually, the final model after two stages is `model/model.pt`.
https://github.com/Project-MONAI/model-zoo/blob/700d1bdd4eca349ab06bd2cc75f1dff8247af911/models/pathology_nuclei_segmentation_classification/configs/train.json#L345-L350